eg/Cli.py

```diff
@@ -149,7 +150,8 @@ class args:
                     if not NamedPipe.send_message('eg.app.Exit, ()'):
                         ctypes.windll.user32.MessageBoxA(
                             0,
-                            'EventGhost cannot restart - unknown error.\n',
+                            'EventGhost cannot restart.             \n\n'
+                            'Unknown Error.                         \n',
```

Comment:
Nope. but in all reality that error should never occur. if it does then there is something that simply does not work right. and I would have no clue what it could be. because the only reason why eg.app.Exit() would return anything but true would be if the user canceled the saving of the changes from a dialog that should not have appeared in the first place,